### PR TITLE
Various feedback fixes / enhancements for first release candidate

### PR DESCRIPTION
**Pull request by blenderer**
- Removes vscode workspace settings from gitignore
- adds tab preferences to vscode settings
- Removes janky loading ellipsis from Accountability View #35
- Removes alert sound from death alert #39
- Fixes alerts to properly use passed config #39
- Remove enable toggle (requested from many users and their feedback)

### CODE REVIEW

File: Hardcore.lua [hunk 1]
```diff
@@ -643,24 +628,21 @@ function Hardcore:Debug(msg)
...
 -- Alert UI
-function Hardcore:ShowAlertFrame(style, message)
-	-- style is a key-based property of ALERT_STYLES
+function Hardcore:ShowAlertFrame(styleConfig, message)
 	-- message is any text accepted by FontString:SetText(message)
 
 	message = message or ""
 
-	local frame, text, icon, file, filename, delay = nil, nil, nil, nil, nil, nil
-	local data = ALERT_STYLES[style] or ALERT_STYLES["hc_red"]
```
Comment (blenderer) [inline Hardcore.lua:653]: This line was always defaulting to the default `ALERT_STYLES["hc_red"]`

Comment (zdeyn) [reply Hardcore.lua:653]: If it works, great - thanks for the fix

**Review by zdeyn — APPROVED** [review]
some good fixes for day 1, can revisit other things afterwards

**Comment by RobertB4** [comment]
Does removing the "Enable" button not result in having an empty space? Eventually we should probably adjust the button positions, right?

**Comment by zdeyn** [comment]
Can we just leave it alone as-is? it's not broken, and users are divided over whether being able to disable/enable it is a good thing. Another UI is on it's way - pause any decisions or thoughts on this until later?

**Comment by RobertB4** [comment]
Leaving the enable/disable feature in defeats the purpose of the addon. What would leaving it in do for us?

**Comment by zdeyn** [comment]
> What would leaving it in do for us?

It allows users to carry out all authorized actions without them having to go to the AddOns config in WoW and uncheck 'load addon'. There is a higher likelihood of them losing footage if they have to manually re-enable it, as opposed to the easier-to-reenable/automatically-reenables after 30sec/reenables on next reload options we can implement

People have legitimate reasons for turning off the addon - we don't have to understand them all, just facilitate them

**Comment by zdeyn** [comment]
Another possible solution is that 'disabling' it means 'turn off all the things that prevent me from doing things I want to do'.. It can still record their actions, it's just 'taking the safety off'

**Comment by RobertB4** [comment]
> It allows users to carry out all authorized actions without them having to go to the AddOns config in WoW and uncheck 'load addon'.

Players are expected to have the addon on at all times, that's the whole point of the addon.

**Comment by zdeyn** [comment]
> Players are expected to have the addon on at all times, that's the whole point of the addon.

And yet, with the addon enabled, they cannot perform some authorized actions - catch22. One way or the other, the user will resolve this as they see fit - either disabling to addon to perform the action, or being limited in their actions because the addon won't allow it.

So, solve the problem from another angle - have a "safety switch", which continues logging all actions, but disables all 'you can't do that in Hardcore' preventions? Then the user can perform the actions, and the addon is still loaded - ready to be re-armed at the click of a button.

The validation string will reveal all, with no chance of cheating

**Comment by zdeyn** [comment]
I'll also add that it's probably the simplest way to let everyone get what they want, with minimal coding required to deliver it

**Comment by aaronma37** [comment]
When i first tried hardcore i remember worrying about whether or not the addon was actually enabled. I think I actually played the first few levels with it off by mistake lol. I do think that if we leave it in we should call it something else (record mode?), so that major things are still done (death, gchat abnouncments) but small things are allowed like opening mailboxes. Though I think its straightforward to just remove since its pretty rare to actually need it off

**Comment by blenderer** [comment]
@RobertB4 adjust the button positions to compensate for the missing button:
image

**Review by RobertB4 — APPROVED** [review]
👍

**Pull request merged**